hedera-node/hedera-mono-service/src/main/java/com/hedera/node/app/service/mono/stats/ServicesStatsConfig.java

```diff
@@ -29,7 +29,7 @@ public final class ServicesStatsConfig {
 
     static final String COUNTER_HANDLED_NAME_TPL = "%sHdl";
     static final String COUNTER_RECEIVED_NAME_TPL = "%sRcv";
-    static final String COUNTER_DEPRECATED_TXNS_NAME = "DeprTxnsRcv";
+    public static final String COUNTER_DEPRECATED_TXNS_NAME = "DeprTxnsRcv";
     static final String COUNTER_ANSWERED_NAME_TPL = "%sSub";
```

Comment:
I didn't see this anywhere else in this PR, is this intentional?

Reply:
Not sure, why this was changed. Reverted the change.